pyfranca/ast.py

```diff
@@ -339,6 +345,15 @@ def __init__(self, name, key_type, value_type):
         self.value_type = value_type
 
 
+class Constant(ComplexType):
```

Comment:
I redesign the ast.py a bit. I add a base class "value" with specialisations integer_value, double_value etc. So pyfranca can handle all kind of values independend of their usage. Currently they are used only for constants.  A "value" is defined by it's nummerical value and type. Type could be integer_value, double_value etc.  

I adjust lexer and parser. I do not adjust the processor, so all checks of type compatibility are missing. Before I start checking the constants, please take a look into my changes. If you like this approach, I will implement the processor part. If not we have to find a better solution.

I rename the lexer token "INTEGER" to "INTEGER_VAL". I see that currently the Franca primitive type "Integer" is not supported. Especially Integer with Range limits. Later when the Integer type is implemented, it is necessary to add the Keyword "Integer" or to use "Integer" as a token. Now we have a clear seperation between Types and Values.


I hope you appreciate my changes.